gpii/node_modules/matchMakerFramework/src/MatchMakerUtilities.js

```diff
@@ -175,6 +188,135 @@ var fluid = fluid || require("infusion"),
         return togo;
     };
 
+    // true if one of the values matches across the two arrays (ignoring the indexing)
+    gpii.matchMakerFramework.utils.containsSameValue = function (arr1, arr2) {
+        var obj1 = {};
+        var i;
+        // convert arr1 to object for fast and easy comparison
+        for (i in arr1) {
+            obj1[arr1[i]] = true;
+        }
+        // compare:
+        for (i in arr2) {
```

Comment:
This can then be a single fluid.find iterating over the 1st hash, indexing into the 2nd hash